### ISSUE TITLE
Cannot show additional slots

### ISSUE THREAD

**Issue by RDelorier** [PeachScript/vue-infinite-loading#24]
version: 2.0.0-rc.2

Unable to show `no-results`,`no-more` slots.
It looks like this bug was caused when fixing #22.

maybe a better fix would be to wrap the spinner in a template ie:

```html
<div class="infinite-loading-container">
    <template v-show="isLoading">
      <slot name="spinner">
        <i :class="spinnerType"></i>
      </slot>
    </template>
...
```

**Comment by PeachScript** [comment]
Oh, it's my fault, I've fixed it in `v2.0.0-rc.3`, please upgrade and try it again,  thanks for your report!

**Comment by RDelorier** [comment]
@PeachScript perfect, thanks for the quick update
